### ISSUE TITLE
shopId not available when fixtures are loaded

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2313]
### Expected behavior
Shop should be loaded before all other fixtures so that every record that requires a `shopId` (which is almost all of them) can use it

### Actual Behavior
Fixture data is getting loaded with a `null` for `shopId` resulting in numerous errors since these records will never be found by the application

One example is #2180 

### Steps to Reproduce the Behavior
This is the hard part, it appears this error only occurs when running on a remote Docker container (at least I have never been able to reproduce it running a local container).

There have been several attempts to fix this issue but they have all been unsuccessful, which is compounded by the fact that it's hard to verify the bug is actually fixed.

**Comment by jshimko** [comment]
I'm pretty sure this shouldn't be possible after my fix in #2224 (which is merged into `development` now).  The shop fixture is the [first thing to load](https://github.com/reactioncommerce/reaction/blob/development/server/startup/load-data.js#L11-L17) on app startup ([before Reaction.init()](https://github.com/reactioncommerce/reaction/blob/development/server/startup/init.js#L9-L12)) and then loading translation fixtures is [completely blocked](https://github.com/reactioncommerce/reaction/blob/development/server/api/core/core.js#L18-L22) until a shopId exists (translations are loaded in the `onCoreInit` hook).

So I guess we just need to confirm that anyone that's still experiencing this issue has tried the development branch in the last 13 days.

**Comment by jshimko** [comment]
Are we sure this has continued to happen after #2224 was merged?  I've never been able to reproduce it, so I'm not sure what to do about it at this point.  

I'm also not sure how it's even possible for other fixtures to load before the shopId exists.  The fixtures are the [first thing that happens](https://github.com/reactioncommerce/reaction/blob/development/server/startup/init.js#L9-L10) on app startup and all secondary fixtures are [blocked until the shopId is available](https://github.com/reactioncommerce/reaction/blob/development/server/startup/load-data.js#L20-L24).  And then `Reaction.init()` is [also blocked](https://github.com/reactioncommerce/reaction/blob/development/server/api/core/core.js#L18-L21) until they're done loading.  I just don't see how it's still possible without Mongo returning inconsistent results.

**Comment by zenweasel** [comment]
For now I am going off the assumption that #2224 has fixed this problem and we can reopen when it's reported again. I would still like to see `hello.reactioncommerce.com` updated and see if it's still happening there.
